### PR TITLE
add convert_zsh_to_array

### PR DESCRIPTION
**Pull request by dirk-thomas**
Connect to colcon/colcon-core#55.

### CODE REVIEW

File: colcon_zsh/shell/template/package.zsh.em [hunk 1]
```diff
@@ -24,8 +24,19 @@ _colcon_package_zsh_source_script() {
   fi
 }
 
+# function to convert array-like strings into arrays
+# to wordaround SH_WORD_SPLIT not being set
```
Comment (mjcarroll) [inline colcon_zsh/shell/template/package.zsh.em]: word->work

File: colcon_zsh/shell/template/prefix.zsh.em [hunk 2]
```diff
@@ -10,6 +10,16 @@ else
   _colcon_prefix_zsh_COLCON_CURRENT_PREFIX="$COLCON_CURRENT_PREFIX"
 fi
 
+# function to convert array-like strings into arrays
+# to wordaround SH_WORD_SPLIT not being set
```
Comment (mjcarroll) [inline colcon_zsh/shell/template/prefix.zsh.em]: word->work

File: colcon_zsh/shell/template/prefix.zsh.em [hunk 3]
```diff
@@ -27,6 +37,8 @@ _colcon_prefix_zsh_prepend_unique_value() {
   IFS=":"
   # start with the new value
   _all_values="$_value"
+  # wordaround SH_WORD_SPLIT not being set
```
Comment (mjcarroll) [inline colcon_zsh/shell/template/prefix.zsh.em]: word->work

**Review by mjcarroll — APPROVED** [review]
Same typo...

**Comment by dirk-thomas** [comment]
Good catch. All three fixed in a0eed1109f64b05d65e63e21dc331c95ed124135.

**Pull request merged**